samples/unit-tests/indicator-chaikin/recalculations/demo.js

```diff
@@ -0,0 +1,174 @@
+
```

Comment:
In a few places, you use chart.series[0], chart.series[1], chart.series[2]. Consider to use variables.